cms/templates/js/add-xblock-component-menu.underscore

```diff
@@ -10,20 +10,39 @@
             <% for (var i = 0; i < templates.length; i++) { %>
                 <% if (!templates[i].boilerplate_name) { %>
                 <li class="editor-md empty">
-                    <button type="button" class="button-component" data-category="<%= templates[i].category %>">
-                        <span class="name"><%= templates[i].display_name %></span>
+                    <button type="button" class="button-component" data-category="<%- templates[i].category %>">
+                        <%= HtmlUtils.HTML(support_template({support_level: templates[i].support_level})) %>
+                        <span class="name"><%- templates[i].display_name %></span>
                     </button>
                 </li>
                 <% } else { %>
                 <li class="editor-md">
-                    <button type="button" class="button-component" data-category="<%= templates[i].category %>"
-                       data-boilerplate="<%= templates[i].boilerplate_name %>">
-                        <span class="name"><%= templates[i].display_name %></span>
+                    <button type="button" class="button-component" data-category="<%- templates[i].category %>"
+                       data-boilerplate="<%- templates[i].boilerplate_name %>">
+                        <%= HtmlUtils.HTML(support_template({support_level: templates[i].support_level})) %>
+                        <span class="name"><%- templates[i].display_name %></span>
                     </button>
                 </li>
                 <% } %>
             <% } %>
         </ul>
-        <button class="cancel-button" data-type="<%= type %>"><%= gettext("Cancel") %></button>
+        <button class="cancel-button" data-type="<%- type %>"><%- gettext("Cancel") %></button>
+        <span class="support-documentation">
```

Comment:
This will be moving into its own template, and I need to handle the following:

1. Don't show any of this of StudioXBlockConfigurationFlag is disabled
2. Pass in the platform name
3. Unfortunately are are stuck hard-coding the readthedocs link for now. :(